docs/api-docs/cmds.md

```diff
@@ -357,6 +389,16 @@ function reducer(state = { /* ... */ }, action) {
 }
 ```
 
+## Convenience helpers
+
+### `Cmd.clearTimeout(timerId)`
+Cancel's a delayed command execution created by `Cmd.setTimeout()`.
+Equivalent to `Cmd.run(clearTimeout, { args: [timerId] })`.
```

Comment:
I don't think this is true. Because of issues with the `this` param and how clearTimeout is called. I've needed to create a wrapper function that calls `clearTimeout` rather than passing it right to Cmd.run

Reply:
The TypeError you mentioned is fixed by #242. You can now pass `createTimeout` directly to `Cmd.run()` and it works as expected.

Reply:
ah forgot about that. nvm then